[DM-3831] Calendar cut in logs tab (#2312)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-2-10-logs-device-tab-dropdown-fixes.md

---
date: ""

> title: Logs device tab dropdown fixes

Reviewer: ```suggestion
title: Fixed layout issues in the Logs tab in the device details
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3831
version: 1020.2.10
---

> Several visual issues found on Device management Logs device tab. These issues were fixed by properly aligning labels, adjusting date and time pickers width and adding proper visual background separation when "type of log" dropdown is open.

Reviewer: ```suggestion
The **Logs** tab in the device details had several layout issues. These issues have been fixed by properly aligning labels, adjusting the date and time pickers width, and adding a proper background separation when the **Type of log** dropdown is open.
```
